FMUv3_REV_D/README.md

```diff
@@ -0,0 +1,11 @@
+# FMUv3 Pixhawk Open Hardware Design
```

Comment:
Placeholder, just copied from old readme.

@LorenzMeier Should the folder be renamed from FMUv3_REV_D to FMUv3 - possibly having the REV as a sub folder?